src/Jeffijoe.MessageFormat/Formatting/Formatters/PluralFormatter.cs

```diff
@@ -293,6 +293,70 @@ private void AddStandardPluralizers()
                     // ReSharper restore CompareOfFloatsByEqualityOperator
                     return "other";
                 });
+            
+            this.Pluralizers.Add(
+                "ru",
+                n =>
+                {
+                    // ReSharper disable CompareOfFloatsByEqualityOperator
+                    if (n == (int) n) 
+                    // ReSharper restore CompareOfFloatsByEqualityOperator
+                    {
+                        var integer = (int) n;
+                        var remainderTen = integer % 10;
+                        var remainderHundred = integer % 100;
+
+                        if (remainderTen == 1 && remainderHundred != 11)
+                        {
+                            return "one";
+                        }
+
+                        if (remainderTen >= 2 && remainderTen <= 4 && (remainderHundred < 12 || remainderHundred > 14))
+                        {
+                            return "few";
+                        }
+
+                        if (remainderTen == 0 || (remainderTen >= 5 && remainderTen <= 9) ||
+                            (remainderHundred >= 11 && remainderHundred <= 14))
+                        {
+                            return "many";
+                        }
+                    }
+
+                    return "other";
+                });
+            
+            this.Pluralizers.Add(
```

Comment:
uk and ru rules are incidentally the same, but decided not to reuse the function, as they may be refactored to multiple files later